bench-tps/src/perf_utils.rs

```diff
@@ -12,12 +12,11 @@ use {
     },
 };
 
+#[allow(dead_code)]
```

Comment:
If we stripped out the unused field, we should no longer need this right ?

Reply:
ah... sorry. let me remove it now 😭

Reply:
fixed: [772b9be](https://github.com/anza-xyz/agave/pull/1346/commits/772b9be56c2061733767aca8c79312b13a6a4af5)